Update tooling for 2023

cadin opened the pull request:
Hi Alex,
Not sure if you're interested in trying to keep this up to date, but I helped a friend get this running again recently, and thought I'd offer the changes back to your project. No worries if you don't want to pull this in for any reason.

- The main change is to now include the Framer library.  
I couldn't figure out how to get things updated in a way that would still build the latest. But since the library won't get updates anymore, I just included the built version in the project.

- With that change, we also no longer need for `yarn`, so I removed that.

- I updated `webpack`, `webpack-dev-server`, and `webpack-cli`, and updated other out-of-date dependencies (`npm audit fix`).

- Some minor edits to the README to reflect the changes, and to update some broken links with archived pages.

Commit:
update project config to resolve build errors

Commit:
update README

Commit:
update README; tagline

alexchantastic commented:
👋 Hi @cadin!

Really happy to see that someone is getting use out of this project after so many years! I would be more than happy to merge this in.

I'm not seeing an included version of `framer.js` in your branch though. Perhaps it wasn't included because the `.gitignore` file at the root of the project includes `src/framer/framer.js`.

Commit:
don’t ignore Framer library

cadin commented:
Aha! You're right. I removed that from the gitignore now.

It's surprisingly hard to find a lightweight, code-first prototyping tool in 2023. I've moved on to just prototyping in straight React now, but that's not exactly lightweight, and is a lot to ask for most designers. For folks who are used to prototyping in Framer Classic or Flash, this project is a nice way to bang out a quick idea or two. Thanks for keeping it alive!

alexchantastic commented:
Thanks for the fix. I'll merge this in now!

Agree on your points about a lightweight, code-first prototyping tool these days. I have also moved to React, but I focus primarily on web applications so it makes a lot of sense for me. If I were more in the mobile design space, I could see more of a need for a toolchain like (old) Framer.

Maybe we could reach out to the Framer folks to see if they could re-host the docs as well. Relying on a cached version on archive.org seems like a painful experience.
